### DISCUSSION TITLE
Is possible to reopen closed in the past topic?

### DISCUSSION THREAD

**Discussion by panciunio** [arendst/Tasmota#18445] [Q&A] [answered]
Hi,

To avoid duplicates of issues I wrote some information into:
[ Power Meter - SDM630 & additional values #6878 ](https://github.com/arendst/Tasmota/issues/6878)

Is possible to reopen existing but closed topic?

Regards,
panciounio

**Comment by Jason2866** [comment, marked as answer]
Closed issues are solved. So open a new issue in Github. You can reference to the closed one. BUT every opened issue needs to have ALL Infos asked in the template.
Discussions should not be closed, if done we can reopen.
